.github/workflows/release.yml

```diff
@@ -33,11 +33,26 @@ jobs:
           FORCE_COLOR: 1
           INFURA_KEY: ${{ secrets.TEST_INFURA_KEY }}
 
-      - name: Update package versions
-        # ${GITHUB_REF##*/} is the branch name
+      - name: Set TAG for master to latest
+        if: ${{ github.ref == 'refs/heads/master' }}
         run: |
-          $(npm bin)/lerna version prerelease --no-git-tag-version --no-push --yes --exact --preid ${GITHUB_REF##*/}
-          git add .
+          echo "TAG=latest" >> $GITHUB_ENV
+
+      - name: Set TAG for non-master to the branch name
+        if: ${{ github.ref != 'refs/heads/master' }}
+        run: |
+          echo "TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: Update package versions for latest release (master)
+        if: ${{ env.TAG == 'latest' }}
```

Comment:
since after `if: ${{ github.ref == 'refs/heads/master' }}` we set the tag to `'latest'`, can't we just remove this if statement and include the following `run: $(npm bin)/lerna version patch --no-git-tag-version --no-push --yes --exact` inside of it?

Not particularly important, but could be a slight improvement.

Reply:
Yeah, we could do that, and avoid these weird `if: `s would be kinda nice. The "step" would just be doing two things now. It'd be named something like "Set TAG for master to latest and update versions for master".